### PR TITLE
Improvements to installation instruction

### PR DESCRIPTION
**Pull request by mattias-p**
I made some improvements to the installation instruction for Debian while testing.

### CODE REVIEW

**Comment by sandoche2k** [comment]
@mattias-p we could approve this only if we test it on Debian. So the approval may take time.

**Comment by mattias-p** [comment]
@sandoche2k I hope this doesn't invalidate too much of your testing.

File: docs/Installation.md [hunk 1]
```diff
@@ -544,7 +544,7 @@ you should be able to use the
 API on localhost port 5000 as below. The command requires that `curl` is installed.
 
 ```sh
-curl -s -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
+wget -q -O - --header "Content-Type: application/json" --post-data '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
```
Comment (matsduf) [inline docs/Installation.md]: Please give a strong argument to change from curl to wget at this late stage. I am against the change for v2019.1.

**Comment by matsduf** [comment]
The commit comment is "Use wget instead of curl on Debian", but the change is on OSs.

**Comment by sandoche2k** [comment]
WGET vs CURL
https://daniel.haxx.se/docs/curl-vs-wget.html. 
I do support @matsduf that we may discuss it for next version

**Comment by mattias-p** [comment]
I don't think curl vs wget is a big deal in this case. Feature-wise both meet our needs. Wget comes pre-installed, at least on some platforms, which is convenient. Removed the commit to sidestep the argument.

**Comment by matsduf** [comment]
@mattias-p, the change in installation of dependencies doe not look like an editorial change. Is it? Can you please describe?

**Comment by mattias-p** [comment]
@matsduf Well, as I described in the commit message, I've added some forgotten dependencies,
shifted to install some dependencies from APT instead of CPAN, and
removed some transitive and/or unused dependencies.

**Comment by matsduf** [comment]
Sorry, the Github hides long commit messages and I did not pay attention to the dots. I think it is more visible when repeated in the description of the PR.

**Comment by mattias-p** [comment]
Thanks, @matsduf.

Waiting to merge until it's been tested on Ubuntu.

**Comment by sandoche2k** [comment]
@mattias-p merge?

**Pull request merged**